test/multiverse/lib/multiverse/suite.rb

```diff
@@ -387,10 +391,13 @@ def execute_child_environment(env_index, instrumentation_method)
         log_test_running_process
         configure_before_bundling
 
+        # require_relative('../../../warning_test_helper')
```

Comment:
whoops, I think this got left behind!

Reply:
oh good catch, my bad lol. fixed! 33b68e474abe813c3091a4ab9d703d64b197e5ad